tests/test_definitions/column_map_expectations/expect_column_values_to_be_in_set.json

```diff
@@ -226,7 +226,7 @@
     "tests" : [
       {
         "title": "basic_positive_test_case_datetime_set",
-        "suppress_test_for": ["sqlite"],
+        "suppress_test_for": ["sqlite", "trino"],
```

Comment:
Don't suppress this datetime test for Trino, as we will be working with their engineers to get this to work